apolloschurchapp/src/content-single/getContentItem.js

```diff
@@ -10,5 +10,5 @@ export default gql`
       }
```

Comment:
You'll need to rename the fragment referenced on line 9 to `contentCardFragment`